frontend/integration-tests/tests/kubevirt/vm.actions.scenario.ts

```diff
@@ -214,4 +216,33 @@ describe('Add/remove disks and NICs on respective VM pages', () => {
 
     expect((searchYAML(networkInterface.networkDefinition, vm.name, vm.namespace, 'vmi'))).toBe(false);
   }, VM_ACTIONS_TIMEOUT * 2);
+
+  it('NIC cannot be added twice using one net-attach-def', async() => {
+    await vm.navigateToTab(TABS.NICS);
+    if ((await vm.getAttachedNics()).filter(nic => nic.name === networkInterface.name).length === 0) {
+      await vm.addNic(networkInterface);
+    }
+
+    // Verify the NIC is added in VM Manifest
+    const resource = getResourceJSON(vm.name, vm.namespace, vm.kind);
+    const nics = _.get(resource, 'spec.template.spec.domain.devices.interfaces').filter(nic => nic.name === networkInterface.name);
```

Comment:
`.find` is semantically closer 

\+ it might be good to add `[]` as a default value in case interfaces field is missing (not sure if that can happen though)

Reply:
Updated.
It shouldn't happen without the code failing before reaching this line, but it doesn't hurt to keep it defensive.